client/components/cart.js

```diff
@@ -1,123 +1,21 @@
-import { getCartFIXME } from '../store';
-import { Link } from 'react-router-dom';
+import React from 'react';
 
-class Cart extends Component {
-  componentDidMount() {
-    // this.props.getCartFIXME();
-  }
-  render() {
-    // const cart  = this.props.cart;
-    // const boats = cart.boats
-
-    const initialBoats = [
-      {
-        id: 1,
-        name: 'Pearl',
-        imageUrl: null,
-        description: '',
-        cost: 815750,
-      },
-      {
-        id: 3,
-        name: 'Forever Young',
-        imageUrl: null,
-        description: '',
-        cost: 359150,
-      },
-      {
-        id: 4,
-        name: 'Squid Pro Quo',
-        imageUrl: null,
-        description: '',
-        cost: 938932,
-      },
-      {
-        id: 5,
-        name: 'Second Chance',
-        imageUrl: null,
-        description: '',
-        cost: 513856,
-      },
-      {
-        id: 7,
-        name: 'Pegasus',
-        imageUrl: null,
-        description: '',
-        cost: 251078,
-      },
-      {
-        id: 8,
-        name: 'Why Not',
-        imageUrl: null,
-        description: '',
-        cost: 235208,
-      },
-      {
-        id: 9,
-        name: "Feelin' Nauti",
-        imageUrl: null,
-        description: '',
-        cost: 482434,
-      },
-      {
-        id: 10,
-        name: 'High Maintenance',
-        imageUrl: null,
-        description: '',
-        cost: 667391,
-      },
-      {
-        id: 11,
-        name: 'Aquaholic',
-        imageUrl: null,
-        description: '',
-        cost: 507032,
-      },
-    ];
-
-    const BOATS_TO_MAKE = 25;
+// cart should be a stateless, functional component like grocery item in add groceries
+const Cart = props => {
+  const cart = props.cart || JSON.parse(window.localStorage.getItem('cart'));
 
-    const boats = [];
-
-    for (let i = 0; i < BOATS_TO_MAKE; i++) {
-      const boat = initialBoats[i % initialBoats.length];
-      boat.id = boat.id + i;
-      boat.name = boat.name + i;
-      boats.push(boat);
-    }
+  console.log('Cart component received cart', cart);
 
+  if (cart) {
     return (
       <div>
-        <ul>
-          {boats.map(boat => {
-            return (
-              <li key={boat.id}>
-                <Link to={`/boats/${boat.id}`}>
-                  <p>{boat.name}</p>
-                </Link>
-                <img src={boat.imageUrl} width="190" height="190" />
-                <p>Cost: {boat.cost}</p>
-              </li>
-            );
-          })}
-        </ul>
+        <h3>Status: {cart.status}</h3>
+        <h3>Total: $ USD {cart.total}</h3>
       </div>
     );
+  } else {
+    return 'no cart';
   }
-}
-
-const mapState = state => {
-  console.log(state);
-  return {
-    // cart: state.cart,
-  };
-};
-const mapDispatch = dispatch => {
-  return {
-    // getCart: () => dispatch(getCartFIXME()),
-  };
 };
 
-export default connect(mapState, mapDispatch)(Cart);
+export default Cart;
```

Comment:
if Cart is going to read from the Redux store down the line, it will need connect